examples/complete-vpc/main.tf

```diff
@@ -1,20 +1,28 @@
 provider "aws" {
-  region = "eu-west-1"
+  region = local.region
```

Comment:
I just hit this issue when using Terraform 0.15.0 - https://github.com/hashicorp/terraform-provider-aws/issues/8040#issuecomment-822774525 

Works well when I put a value there as a string:
```
region = "eu-west-1"
```